packages/mdc-radio/adapter.ts

```diff
@@ -21,35 +21,20 @@
  * THE SOFTWARE.
  */
 
-/* eslint-disable no-unused-vars */
-import {MDCSelectionControlState} from '@material/selection-control/index';
-
-/* eslint no-unused-vars: [2, {"args": "none"}] */
-
 /**
  * Adapter for MDC Radio. Provides an interface for managing
- * - classes
- * - dom
- *
- * Additionally, provides type information for the adapter to the Closure
- * compiler.
  *
  * Implement this adapter for your framework of choice to delegate updates to
  * the component in your framework of choice. See architecture documentation
  * for more details.
  * https://github.com/material-components/material-components-web/blob/master/docs/code/architecture.md
  *
- * @record
  */
-class MDCRadioAdapter {
-  /** @param {string} className */
-  addClass(className) {}
-
-  /** @param {string} className */
-  removeClass(className) {}
 
-  /** @param {boolean} disabled */
-  setNativeControlDisabled(disabled) {}
+interface MDCRadioAdapter {
+  addClass(className: string): void;
+  removeClass(className: string): void;
+  setNativeControlDisabled(disabled: boolean): void;
 }
 
-export default MDCRadioAdapter;
+export {MDCRadioAdapter as default, MDCRadioAdapter};
```

Comment:
We should do a pass over the already-converted TypeScript files and make sure we're using this same pattern.

Reply:
Done